src/main/java/com/example/controller/GraphEditController.java

```diff
@@ -45,10 +56,15 @@ private void initializeDrawingButtons() {
     }
 
     private void initializeSimulationButtons() {
+        startButton.setOnAction(e -> simulationController.initSimulation());
```

Comment:
These buttons should call methods form its parent controller (in this case SimulationMenuController). In SimulationMenuController equivalent buttons have already defined actions. We could for example make public buttons in SimulationMenuController and call button.fire() in startButton.setOnAction().